src/main/resources/archetype-resources/project-base/src/main/java/wat/base/AbstractBaseTest.java

```diff
@@ -12,11 +12,11 @@
  */
 public class AbstractBaseTest<P extends ProjectBaseTest> extends ProjectBaseTest<P> {
```

Comment:
This could be confusing, having AbstractBaseTest and ProjectBaseTest. Wouldn't it be simpler if we kept one of them?